src/app/components/dashboard-apps-dock/dashboard-apps-dock.less

```diff
@@ -0,0 +1,304 @@
+    left: 50%;
+    top: 100%;
+    z-index: 1000;
+
+    // Temp / To Review
+    margin-top: 68px;
+    // display: none;
+
+    visibility:hidden;
+    opacity:0;
+    transition: visibility 0s linear 0.5s,opacity 0.5s linear;
+
+    div.dock-arrow {
+      position: relative;
+      right: 50%;
+    }
+
+    div.inner-dock-menu { // Inner div holding the menu
+      position: relative;
+      right: 50%;
+      padding: 5px 0px;
+      .border-top-radius(5px);
+      .border-bottom-radius(5px);
+      background-color: @bg-inverse-color;
+
+      .box-shadow(0 6px 12px rgba(0, 0, 0, 0.175));
+
+      .menu-item {
+        cursor: pointer;
+        padding: 5px 20px;
+
+        &:hover {
+          color: white !important;
+          .border-top-radius(5px);
+          .border-bottom-radius(5px);
+
+          background-color: lighten(@decorator-alt-color,10%);
+        }
+
+        a {
+          color: darken(@text-inverse-color,25%);
+          display: flex;
+          text-decoration: none;
+
+          &.connect-btn-center, img.connect-btn-center {
+            margin: auto;
+          }
+
+          i.fa {
+            color: @decorator-inverse-color;
+            margin-top: 3px;
+            margin-right: 6px;
+          }
+        }
+
+        .connector-logo {
+          margin-top:3px;
+          margin-bottom:3px;
+          cursor: pointer;
+        }
+      }
+    }
+  }
+
+  & > li.active {
+    .dock-menu {
+      // display: block;
+      visibility: visible;
+      opacity: 1;
+      transition-delay: 0s;
+    }
+  }
+}
+
+ul.dock.minimized {
+  height: @mno-dock-app-logo-square-width;
+  .transition(height 0.3s);
+
+  img.app-logo, .add-app {
+    .square(@mno-dock-app-logo-minimized-square-width);
+  }
+
+  .add-app {
+    span {
+      .transition(all 0.3s);
+      display: none;
+    }
+  }
+
+  .dock-menu {
+    margin-top: @mno-dock-app-logo-minimized-square-width;
+  }
+}
+
+ul.dock.animated  > li:hover, ul.dock > li.active {
+  img.app-logo {
+    transform: scale(1.8) translate(0, 10px);
+    margin: 5px 1.6em 5px;
+    z-index: 9999;
+
+    border: 1px solid rgba(0, 0, 0, 0.15);
+    .box-shadow(0 6px 12px rgba(0, 0, 0, 0.175));
+  }
+
+  .title {
+    display: block;
+  }
+
+  /* one element after and element before (with JS)*/
+  & + li img.app-logo, ul.dock > li.prev img.app-logo {
+    .scale(1.4);
+    margin: 5px 1.6em 5px;
+  }
+}
+
+ul.dock  > li.prev img.app-logo {
+  .scale(1.4);
+  margin: 5px 1.4em 5px;
+}
+
+// Mobile
+#app-dock-mobile-container {
+  position: fixed;
+  bottom: 0;
+  width: 100%;
+  z-index: 1030;
+  color: white;
+
+  .mobile-dock {
+    width: 100%;
+    text-transform: uppercase;
+    padding: 10px;
+    background-color: @purple;
+  }
+}
+
+#mobile-dock {
+  background-color: @darkblue;
+  z-index: 1030;
+
+  &.collapse.in {
+    overflow-y: auto;
+  }
+
+  .btn {
+    text-transform: uppercase;
+  }
+
+  ul > li {
+    border-bottom: 1px solid @mediumblue;
```

Comment:
Can be a less variable. Also be careful, as @mediumblue is only defined in impac-angular. If impac-angular changes it color variables this would break.